.github/workflows/build_and_test.yml

```diff
@@ -977,13 +977,13 @@ jobs:
           mkdir -p extensions-to-upload
 
           docker cp ${{ steps.create-container.outputs.EID }}:/extensions/ ./extensions-to-upload/
+          docker cp ${{ steps.create-container.outputs.EID }}:/ext_index.json ./extensions-to-upload/
 
           # Remove control files, they'll be replaced by an index file
-          rm -f ./extensions-to-upload/*.control
+          # TODO: this doesn't actually work because control files are in subdirectories
+          rm -f ./extensions-to-upload/extensions/*.control
```

Comment:
let's fix this?

this should work:
`find  ./extensions-to-upload/extensions/ -name '*.control' | xargs rm`

Reply:
yes let's get rid of these files